advisor_engine/endpoints.py

```diff
@@ -117,8 +118,39 @@ def handle_api_ping():
     return Response(status_code=status.HTTP_200_OK)
 
 
+class LimitRequestBodyMiddleware(BaseHTTPMiddleware):
```

Comment:
I don't think you need a middleware for this.  Should be built into fast api
`app = FastAPI(max_request_size=<bytes>)`

Reply:
Going to close this comment. I don't believe `max_request_size` is a valid config option.